runtime/common/src/lib.rs

```diff
@@ -199,6 +199,13 @@ pub mod constants {
 	pub const CENTI_CFG: Balance = 10 * MILLI_CFG; // 10−2 	0.01
 	pub const CFG: Balance = 100 * CENTI_CFG;
 
+	// Native tokens' decimals
```

Comment:
By 'native' I am referring to tokens supported natively, hardcoded in the runtime code. Happy to use another naming here, just suggest an alternative.